app/lib/buttonsAndButtonGroups.ts

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import * as VexFlow from "vexflow";
+const { Renderer } = VexFlow.Flow;
+import { GlyphProps } from "./typesAndInterfaces";
+import {
+  KeySigAction,
+  KeySigState,
+  NoteInteractionAction,
+  NoteInteractionState,
+  StaveNoteData,
+} from "./typesAndInterfaces";
+import { initializeRenderer } from "./initializeRenderer";
+const modifyNotesActionTypes = {
```

Comment:
Another small thing, but I think it can be helpful to add empty lines once in a while between things like import statements and const's for example, or other sections in the code, just for added readability